packages/mdl-menu/simple/mdl-simple-menu.scss

```diff
@@ -141,12 +141,22 @@ $mdl-simple-menu-item-offset: 10px;
     text-decoration: none;
 
     @include mdl-typography(body2);
+
+    @include mdl-theme-dark(".mdl-simple-menu", true) {
```

Comment:
qq: why pass `true` here and below? Since these selectors aren't compound selectors.